components/core/src/clp_s/ArchiveConstants.hpp

```diff
@@ -0,0 +1,14 @@
+#ifndef CLP_S_ARCHIVE_CONSTANTS_HPP
+#define CLP_S_ARCHIVE_CONSTANTS_HPP
+
+namespace clp_s::constants {
+constexpr char cArchiveSchemaMapFile[] = "/schema_ids";
```

Comment:
Since we're grouping all of them together, can we alphabetize? Or we could split them into groups and alphabetize within each group.